frb_codegen/src/parser/mod.rs

```diff
@@ -631,6 +633,29 @@ impl Parse for DefaultValues {
     }
 }
 
+#[cfg(feature = "serde")]
+impl serde::Serialize for DefaultValues {
+    fn serialize<S>(&self, s: S) -> std::result::Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        const NAME: &str = "DefaultValues";
+        match self {
+            Self::Str(value) => s.serialize_newtype_variant(NAME, 0, "Str", &value.value()),
+            Self::Bool(value) => s.serialize_newtype_variant(NAME, 1, "Bool", &value.value()),
+            Self::Int(value) => {
+                s.serialize_newtype_variant(NAME, 2, "Int", &value.base10_parse::<i64>().unwrap())
+            }
+            Self::Float(value) => {
+                s.serialize_newtype_variant(NAME, 3, "Float", &value.base10_parse::<f64>().unwrap())
+            }
+            Self::Vec(value) => {
+                s.serialize_newtype_variant(NAME, 4, "Vec", &value.into_iter().collect::<Vec<_>>())
+            }
+        }
+    }
+}
```

Comment:
uber-nit: hmm is it possible we do not write down this many boilerplate, but instead let serde generate for us?

Reply:
No problem, I wrote this so that I wouldn't have to convert it thru a serializable shell struct first.